packages/cyberstorm/src/components/Forms/CreateTeamForm/CreateTeamForm.tsx

```diff
@@ -0,0 +1,118 @@
+"use client";
+import styles from "./CreateTeamForm.module.css";
+import * as Button from "../../Button";
+import { TextInput } from "../../TextInput/TextInput";
+import { useForm, FieldErrors, useController } from "react-hook-form";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faArrowsRotate } from "@fortawesome/free-solid-svg-icons";
+
+import { useContext } from "react";
+import { ToastContext } from "../../Toast/ToastContext";
+import {
+  faCircleCheck,
+  faCircleExclamation,
+  faOctagonExclamation,
+} from "@fortawesome/pro-solid-svg-icons";
+
+/**
+ * Form for creating a team
+ */
+export function CreateTeamForm() {
+  const useToast = () => useContext(ToastContext);
+  const toast = useToast();
+
+  async function onValid(data: { teamName: string }) {
+    // TODO: Add sending to TS API endpoint
+    toast?.addToast(
+      "info",
+      <FontAwesomeIcon icon={faCircleExclamation} />,
+      "Creating team"
+    );
+    await new Promise((resolve) => {
+      setTimeout(resolve, 1000);
+    });
+    // TODO: Add a toast on response return based on the response
```

Comment:
Also remove the "Creating team" toast once the request is complete?

Reply:
The idea is that it goes away after it's timer is done, found it a bit better for UX and there's plenty of room for few toasts. We'll tweak the timers and animations after MVP with Gösta 👍